src/snowcli/utils.py

```diff
@@ -478,66 +477,59 @@ def install_packages(
 
 
 def recursive_zip_packages_dir(pack_dir: str, dest_zip: str) -> bool:
-    # TODO:
-    # 1. why this is -> bool
-    # 2. Reformat it to with open...
-    # 3. Reformat it to get list od dirs and then iterate through it, adding files to zip
-    # create a zip file object
-    zipf = zipfile.ZipFile(dest_zip, "w", zipfile.ZIP_DEFLATED, allowZip64=True)
-
-    # for every file in the relative path pack_dir, add it to the zip file
-    for file in pathlib.Path(pack_dir).glob("**/*"):
-        zipf.write(file, arcname=os.path.relpath(file, pack_dir))
-
-    # zip all files in the current directory except the ones that start with "." or are in the pack_dir
-    for file in pathlib.Path(".").glob("**/*"):
-        if (
-            not str(file).startswith(".")
-            and not file.match(f"{pack_dir}/*")
-            and not file.match(dest_zip)
-        ):
-            zipf.write(os.path.relpath(file))
-
-    if os.getenv("SNOWCLI_INCLUDE_PATHS") is not None:
-        for dir_path in os.getenv("SNOWCLI_INCLUDE_PATHS", "").split(":"):
-            directory = pathlib.Path(dir_path)
-            if directory.is_dir():
-                for file in pathlib.Path(directory).glob("**/*"):
-                    if (
-                        not str(file).startswith(".")
-                        and not file.match("*.pyc")
-                        and not file.match("*__pycache__*")
-                    ):
-                        zipf.write(file, arcname=os.path.relpath(file, directory))
-
-    # close the zip file object
-    zipf.close()
+    files_to_pack = get_list_of_files_to_pack(pack_dir, True)
+
+    with ZipFile(dest_zip, "w", ZIP_DEFLATED, allowZip64=True) as package_zip:
+        for file in files_to_pack:
+            package_zip.write(file[0], arcname=os.path.relpath(file[0], file[1]))
```

Comment:
Do we need to use `file[0]` ? Can't we use `file.name` or something to be more explicite?

Reply:
I added File dataclass - now it's file.name